internal/cloud/atlas/groups.go

```diff
@@ -17,27 +19,67 @@ type Group struct {
 	Name string `json:"name"`
 }
 
-type groupResponse struct {
+// Link contains the url to the next for a set of groups and its current position in the chain
```

Comment:
[meganit] "link" is a much more general concept than what we're specifically using it for here
```suggestion
// Link contains additional data related to the response
```
useful article: https://restfulapi.net/hateoas/